### PR TITLE
Add Go proto build rule for p4runtime

### PR DESCRIPTION
**Pull request by akarshgupta25**
This PR adds go_proto_library rule for p4runtime.proto in BUILD file.

### CODE REVIEW

**Review by smolkaj — APPROVED** [review]
LGTM.

**Comment by smolkaj** [comment]
@antoninbas, could you take a look?
I believe this should be an uncontroversial change that is just adding a missing BUILD target.

**Review by antoninbas — APPROVED** [review]

File: proto/BUILD.bazel [hunk 1]
```diff
@@ -100,6 +91,21 @@ py_proto_library(
...
+    protos = [
+        ":p4info_proto",
+        ":p4types_proto",
+    ],
+)
+
+go_proto_library(
+    name = "p4runtime_go_proto",
+    importpath = "github.com/p4lang/p4runtime/go/p4/v1",
+    protos = [":p4runtime_proto"],
```
Comment (smolkaj) [inline proto/BUILD.bazel:106]: The build is currently failing. I believe we have two include some additional dependencies here:

```
compilepkg: missing strict dependencies:
	/home/runner/.cache/bazel/_bazel_runner/1fc64d2e702c0d08ecdb88fe41750bb3/sandbox/linux-sandbox/2/execroot/com_github_p4lang_p4runtime/bazel-out/k8-fastbuild/bin/p4runtime_go_proto_/github.com/p4lang/p4runtime/go/p4/v1/p4runtime.pb.go: import of "github.com/p4lang/p4runtime/go/p4/config/v1"
	/home/runner/.cache/bazel/_bazel_runner/1fc64d2e702c0d08ecdb88fe41750bb3/sandbox/linux-sandbox/2/execroot/com_github_p4lang_p4runtime/bazel-out/k8-fastbuild/bin/p4runtime_go_proto_/github.com/p4lang/p4runtime/go/p4/v1/p4runtime.pb.go: import of "google.golang.org/genproto/googleapis/rpc/status"
No dependencies were provided.
```

**Comment by antoninbas** [comment]
@akarshgupta25 the Github workflow which checks the bazel build is failing

**Review by smolkaj — CHANGES_REQUESTED** [review]

**Comment by akarshgupta25** [comment]
I am unable to trigger a rebuild on Travis CI.
@antoninbas is there a way I can trigger the build to verify the new changes?

**Review by smolkaj — APPROVED** [review]
Thanks for getting things to build! LGTM.

**Comment by smolkaj** [comment]
@antoninbas I believe this is ready to be merged?

**Comment by smolkaj** [comment]
That was fast, thank you! :)

**Pull request merged**